react/week1/todo/src/ToDo.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const ToDo = ({ title, deadLine }) => {
```

Comment:
```suggestion
export const Todo = ({ title, deadLine }) => {
```

I wonder consider renaming this to keep it consistent with your Todos component.